doc/tutorials/enable_ivshmem.rst

```diff
@@ -71,6 +71,13 @@ For the shared memory region:
 #. Enter a virtual Board:Device.Function (BDF) address for each VM or leave it
    blank. If the field is blank, the tool provides an address when the
    configuration is saved.
+
+   .. note::
+
+      The release v3.0 ACRN Configurator has an issue where you need to save the
```

Comment:
Can you add the Github issue number to the note?

Reply:
Updated